tests/component/_task_modules/test_triggers_properties.py

```diff
@@ -99,3 +109,43 @@ def test___ai_task___reset_uint32_property___returns_default_value(ai_voltage_ta
     del ai_voltage_task.triggers.reference_trigger.pretrig_samples
 
     assert ai_voltage_task.triggers.reference_trigger.pretrig_samples == 2
+
+
+def test___ai_voltage_field_daq_task___get_timestamp_property___returns_default_value(
+    ai_voltage_field_daq_task: Task,
+):
+    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
+    when_value = ai_voltage_field_daq_task.triggers.start_trigger.trig_when
+    assert when_value.lsb == 0
+    assert when_value.msb == 0
```

Comment:
arrange, act, assert
```suggestion
    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
    
    when_value = ai_voltage_field_daq_task.triggers.start_trigger.trig_when
    
    assert when_value.lsb == 0
    assert when_value.msb == 0
```